services/sharedchannel/channelinvite_test.go

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package sharedchannel
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
```

Comment:
Nit: looks a bit odd with so many newlines in between them.

Reply:
Don't know why goimports did that. Will amend

Reply:
amended in https://github.com/mattermost/mattermost-server/pull/16803/commits/749ec630122e22d8f49487d7c88457cbc7ba70f4